tools/benchcomp/benchcomp/visualizers/__init__.py

```diff
@@ -206,12 +223,47 @@ def _eval_column_text(column_spec):
         return column_spec
 
 
+    @staticmethod
+    def _parse_scatterplot_config(scatterplot_config_string):
+        if (scatterplot_config_string is None or
+                scatterplot_config_string == "off"):
+            return Plot.OFF
+        elif scatterplot_config_string == "linear":
+            return Plot.LINEAR
+        elif scatterplot_config_string == "log":
+            return Plot.LOG
+        else:
+            logging.error(
+                "Invalid scatterplot configuration '%s'",
+                scatterplot_config_string)
+            sys.exit(1)
+
+
     @staticmethod
     def _get_template():
         return textwrap.dedent("""\
             {% for metric, benchmarks in d["metrics"].items() %}
             ## {{ metric }}
 
+            {% if scatterplot and metric in d["scaled_metrics"] and d["scaled_variants"][metric]|length == 2 -%}
```

Comment:
Non blocking but any chance these values can be stored separately in a config structure and re-used wherever? In case we want to change values later etc.

Reply:
Which "values" are you specifically referring to?

Reply:
Ah apologies, should have pointed to line 250. I meant the values initialized for the quadrantChart. For example pointRadius: 2. If we could store these values separately, imo it would make them more re-usable. Hope that clarifies.

Reply:
I could put those in the Python code in some other place and make Jinja read that variable if that's what you are asking for? I don't really think that makes much of a difference, though? If you are asking for a configuration file: GitHub does not support this, all plot configuration must be done inline.

Reply:
I was referring to doing jinja read the variable from a python enum, but you're right, not much of a difference. You can resolve this comment.